lib/waitforapp.js

```diff
@@ -64,3 +88,4 @@ function waitForAppSync(client, source, callback) {
 }
 
 module.exports.waitForApp = waitForApp;
+module.exports.waitForActivity = waitForActivity;
```

Comment:
I see that some modules expose more than one method (```getapp``` for example) so I thought it's good idea here as well as it's private module currently. But I'm not sure about conventions you have :) So just let me know what looks better for you!